src/main/java/uk/gov/hmcts/dts/fact/repositories/CourtTypeRepository.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.dts.fact.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import uk.gov.hmcts.dts.fact.entity.CourtType;
+
+import java.util.Optional;
+
+public interface CourtTypeRepository extends JpaRepository<CourtType, Integer> {
+
+    Optional<CourtType> findByName(String name);
```

Comment:
I can't see where this is used. If it is not used you can remove it. It is fine to have an empty interface here. It will just use any default methods provided by the JpaRepository class

Reply:
Removed method from CourtTypeRepository class